OCPBUGS-16019 documenting missed bufg fix in 4.14.0 z-stream (#71694)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

* Previously, for Intel E810 NICs, resetting a MAC address on an SR-IOV with a virtual function (VF) when a pod was deleted caused a failure. This resulted in a long delay when creating a pod with SR-IOV VF. With this update, the container network interface (CNI) does not fail fixing this issue. (link:https://issues.redhat.com/browse/OCPBUGS-5892[*OCPBUGS-5892*])

> * Previously, an issue was observed in the {product-title} with some pods getting stuck in the `terminating` state. This affected the reconciliation loop of the allowlist controller resulting in unwanted retries which caused the creation of multiple pods.

Reviewer: Remove "the" in 1st sentence:
```suggestion
* Previously, an issue was observed in {product-title} with some pods getting stuck in the `terminating` state. This affected the reconciliation loop of the allowlist controller resulting in unwanted retries which caused the creation of multiple pods.
```

> * Previously, an issue was observed in the {product-title} with some pods getting stuck in the `terminating` state. This affected the reconciliation loop of the allowlist controller resulting in unwanted retries which caused the creation of multiple pods.

Reviewer: Suggestion only, but felt like the 2nd sentence needed a break or two or somehow. Maybe something like:
```suggestion
* Previously, an issue was observed in the {product-title} with some pods getting stuck in the `terminating` state. This affected the reconciliation loop of the allowlist controller, which resulted in unwanted retries and caused the creation of multiple pods.
```
Or:
```suggestion
* Previously, an issue was observed in the {product-title} with some pods getting stuck in the `terminating` state. This affected the reconciliation loop of the allowlist controller, which resulted in unwanted retries that caused the creation of multiple pods.
```

> With this update, the allowlist controller only inspects pods that belong to the current daemon set and now no retry occurs when one or more pods are not ready. (link:https://issues.redhat.com/browse/OCPBUGS-16019[*OCPBUGS-16019*])

Reviewer: Suggestion only:
```suggestion
With this update, the allowlist controller only inspects pods that belong to the current daemon set. As a result, retries no longer occur when one or more pods are not ready. (link:https://issues.redhat.com/browse/OCPBUGS-16019[*OCPBUGS-16019*])
```

$ oc adm release info 4.14.0 --pullspecs
----
//replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.

[id="ocp-4-14-0-bug-fixes"]

> ==== Bug fix

Reviewer 2: This would with the `Networking` bug [fixes](https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html#ocp-4-14-bug-fixes:~:text=OCPBUGS%2D11889)-,Networking,-Previously%2C%20when%20a) that shipped with 4.14.0

Author: Sorry I am  confused @JoeAldinger I was asked to add it here as it was not available in 4.14 GA but was fixed in this z stream but did not get mentioned as the security level was set incorrectly.

Reviewer 3: @kquinn1204 the bug text should go with the rest of the GA bug fixes instead of making a new bug fix section. The bug is attached to the 4.14.0 advisory, not a 4.14.z advisory.
